homeassistant/components/swiss_public_transport/config_flow.py

```diff
@@ -98,6 +101,7 @@ async def async_step_import(self, import_input: dict[str, Any]) -> FlowResult:
             )
             return self.async_abort(reason="unknown")
 
+        await self.async_set_unique_id(import_input[CONF_NAME])
```

Comment:
This is different compared to the unique ID above?

Reply:
@frenck  Yaml provided a name for each sensor, with config flow therexis none. For compatibility it has been used as name. Therefore i thought the same applies to the unique id. 

Otherwise I can also switch to the same as for the config flow.

Reply:
> Otherwise I can also switch to the same as for the config flow.

That would make more sense, other the above check wouldn't work. If you store different unique IDs, you'll end up with things that don't match.

Additionally, we don't allow the use of names in unique IDs :)

Reply:
Ok, will change it so they are the same👍

Reply:
Now both the config flow and import create a unique id based on the config values the same way.